sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -54,19 +54,15 @@
 #include "math/pprz_geodetic_int.h"
 #include "math/pprz_isa.h"
 
+#ifndef INS_BARO_AGL_OFFSET
```

Comment:
This is basically a question of coordinate system...
Right now most other parts of Paparazzi assume that if you are on the ground, you are at 0 AGL and not at 0.3m if your baro is mounted 30cm from the landing legs.